platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/pax/web/jetty/FilterInjector.java

```diff
@@ -58,8 +61,12 @@
    * @param filterList filters that should be injected.
```

Comment:
✏️ Replace all direct references to `SecurityJavaSubjectFilter` (above) in comments and code.